src/main/java/com/battleasya/admin360/listener/JoinLeaveEvent.java

```diff
@@ -20,10 +21,11 @@ public void onJoin(PlayerJoinEvent event){
 
         Player player = event.getPlayer();
 
-        if (User.hasPermission(player, Permission.ATTEND_TICKET, false)){
+        if (Config.message_pending_on_join &&
+                User.hasPermission(player, Permission.ATTEND_TICKET, false)){
             Admin.addAdmin(player.getUniqueId());
+            User.messagePlayer(player, Config.pending_tickets.replace("{num}", String.valueOf(Request.requestPending.size())));
         }
```

Comment:
Suggested changes:
```
if (User.hasPermission(player, Permission.ATTEND_TICKET, false)){
    Admin.addAdmin(player.getUniqueId());
    if (Config.message_pending_on_join) {
       	User.messagePlayer(player, Config.pending_tickets.replace(
       	 "<PENDING>", Integer.toString(Request.getPndLstSize())
       	));
    }
}
```
- [Critical] When `message_pending_on_join` is false, admins will not be added to the admin list, depriving them from receiving any admin broadcasts.

- [Optional] Utilise the available class function `Request.getPndLstSize()`.

- [Optional] Use <PENDING> as the message placeholder to better align with the default config.

Reply:
Added all three suggestions.

Reply:
Thanks! Excellent edits!